src/app/app.component.ts

```diff
@@ -10,6 +10,9 @@ import { AppState } from './app.service';
  * Top Level Component
  */
 @Component({
+  host:{
+    class:'app app-component flex-container in-column-direction flex-grow-1'
```

Comment:
This the correct way to style the host element (http://stackoverflow.com/questions/32853924/angular-2-how-to-style-host-element-of-the-component) . host is just simple json ... not a stringified jason fragment ... so "class" or 'class' is not required . What is the reason you want "class" in quotes ?